src/components/views/rooms/MessageComposer.tsx

```diff
@@ -216,9 +216,17 @@ class PollButton extends React.PureComponent<IPollButtonProps> {
                 description: _t("You do not have permission to start polls in this room."),
             });
         } else {
-            Modal.createTrackedDialog('Polls', 'create', PollCreateDialog, {
-                room: this.props.room,
-            }, 'mx_CompoundDialog');
+            Modal.createTrackedDialog(
+                'Polls',
+                'create',
+                PollCreateDialog,
+                {
+                    room: this.props.room,
+                },
+                'mx_CompoundDialog',
+                false,  // isPriorityModal
+                true,   // isStaticModal
```

Comment:
```suggestion
                false, // isPriorityModal
                true,  // isStaticModal
```

two spaces is Pythonic, not JSic/TSic